cdap-cli/src/main/java/co/cask/cdap/cli/completer/supplier/EndpointSupplier.java

```diff
@@ -46,9 +49,10 @@ private EndpointSupplier(final ServiceClient serviceClient, CLIConfig cliConfig)
   public Completer getCompleter(String prefix, Completer completer) {
     if (prefix != null && !prefix.isEmpty()) {
       String prefixMatch = prefix.replaceAll("<.+?>", "<>");
-      if (METHOD_PREFIX.equals(prefixMatch)) {
+      if ((METHOD_PREFIX.equals(prefixMatch) || METHOD_PREFIX_WITH_APP_VERSION.equals(prefixMatch))
```

Comment:
could you add some comments here about what command prefixes these conditions handle?